sklearn/mixture/_bayesian_mixture.py

```diff
@@ -118,13 +118,15 @@ class BayesianGaussianMixture(BaseMixture):
         The number of initializations to perform. The result with the highest
         lower bound value on the likelihood is kept.
 
-    init_params : {'kmeans', 'random'}, default='kmeans'
+    init_params : {'kmeans', 'kmeans++', 'random', 'random_from_data'}, default='kmeans'
         The method used to initialize the weights, the means and the
         covariances.
-        Must be one of::
+        String must be one of:
 
             'kmeans' : responsibilities are initialized using kmeans.
             'random' : responsibilities are initialized randomly.
+            'random_from_data' : Initial means are randomly selected data points.
+            'k-means++' : Use the k-means++ method to initialize.
```

Comment:
Please add a `versionchanged:: 1.1` field saying that this params now accepts 'random_from_data' and 'k-means++'.